src/launcher/java/org/truffleruby/launcher/RubyLauncher.java

```diff
@@ -28,6 +28,7 @@
     private final CommandLineOptions config = new CommandLineOptions();
 
     public static void main(String[] args) {
```

Comment:
@pitr-ch Does this main() always gets called in GraalVM?

Reply:
in Ruby launcher yes. Not in other launchers.

Reply:
OK, sounds good then.